pkg/orchestration/wiring/testdata/edge.provided_upstream.state.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: orchestration.voyager.atl-paas.net/v1
```

Comment:
Looks good, just add some more examples of state.yaml with validation errors (e.g. empty resource spec or empty `upstream_address: []`.